### ISSUE TITLE
Not exactly an issue, but may be worth mentioning

### ISSUE THREAD

**Issue by brunoaugusto** [wilfredwee/photon-australis#55]
After the recent update of Chrome that made it utterly slow, I decided to move back to Firefox, but I never liked the default theme/colours. After searching for a way to make it look a little bit like Chrome, I found this repository.

But after the procedures, however, it looked even worse than before. I was about to give up when I accidentally opened the **Themes** section under Add-ons and found that Mozilla offers three variants of their default theme and after activating the *Light* version, my preference of the code you published, and everything became as sexy as you described.

Long story short, perhaps you may want to update your `readme.md` **reinforcing** (since it's already there) that the corresponding theme must be activated together.

Thank you for your good work

**Comment by wilfredwee** [comment]
Hi there,

Sorry for the late reply, I've been quite busy lately. Thanks for your feedback! I've updated the README to be much more detailed. Please let me know if anything else is confusing/any way to improve the README.
